### PR TITLE
chore: disable add video button and update banner information

### PR DESCRIPTION
**Pull request by ayushsatyam146**
Disabled the Add video button and updated the Banner.

Previous:

image

Current:

image

### CODE REVIEW

**Review by hybridx — APPROVED** [review]
LGTM. @ayushsatyam146 can you please bump up the minor version in package.json

**Review by hybridx — APPROVED** [review]

File: video-library-client/src/App.js [hunk 1]
```diff
@@ -36,9 +36,10 @@ const App = () => {
               style={{
                 color: "#fff",
               }}
-              href="mailto:one-platform-devs@redhat.com"
+              href="https://videos.learning.redhat.com/"
```
Comment (riginoommen) [inline video-library-client/src/App.js:39]: Convert this as an environment variable

File: video-library-client/src/App.js [hunk 2]
```diff
@@ -36,9 +36,10 @@ const App = () => {
...
             >
-              The video library is about to be decommissioned on 31 Dec'22.
+              In response to Media Space (http://videos.learning.redhat.com/)
```
Comment (riginoommen) [inline video-library-client/src/App.js:41]:
```suggestion
              In response to Media Space (http://videos.learning.redhat.com/)
```
Convert this as an environment variable

File: video-library-client/src/App.js [hunk 3]
```diff
@@ -36,9 +36,10 @@ const App = () => {
...
+              the video library is about to be decommissioned on 31 Dec'22.
```
Comment (riginoommen) [inline video-library-client/src/App.js:42]:
```suggestion
              the video library is about to be decommissioned on 31st Dec'22.
```

**Review by riginoommen — CHANGES_REQUESTED** [review]
Please see above changes

**Review by riginoommen — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Pull request merged**